src/stdune/either.mli

```diff
@@ -3,3 +3,5 @@
 type ('a, 'b) t =
   | Left  of 'a
   | Right of 'b
+
+val either : ('a, 'b) t -> l:('a -> 'c) -> r:('b -> 'c) -> 'c
```

Comment:
I would call this function `map`

Reply:
Sure. Though this function is called `either` in Haskell for example.